metricbeat/module/redis/info/info.go

```diff
@@ -95,6 +95,18 @@ The document sent to elasticsearch has the following structure:
 	      "total_connections_received": "146",
 	      "total_net_input_bytes": "2247",
 	      "total_net_output_bytes": "277354"
+	    },
```

Comment:
@ruflin Can we just link redis INFO docs, instead of having this ~100 line example of the ES document?